class Error(Exception):
    pass

class NotebookNotFoundError(Error):
    pass